web-crawler/src/main/java/com/google/sps/webcrawler/RelevancyChecker.java

```diff
@@ -28,23 +28,24 @@
 
 // @TODO [Compute the salience of other features, such as location, election, to improve accuracy.]
 /**
- * Provides a tool for checking the relevancy of news article content to a candidate.
+ * A {@code LanguageServiceClient} that performs entity analysis to check the relevancy of news
```

Comment:
Did you mean `{@code RelevancyChecker}`? (here and below?)

Reply:
Yeah I think RelevancyChecker makes more sense. I'll make the changes

Reply:
On line 42, I still kept the comment as `LanguageServiceClient` because the exception would come from that